src/Surfnet/StepupRa/RaBundle/Controller/VettingController.php

```diff
@@ -123,7 +123,7 @@ public function startProcedureAction(Request $request)
             return ['form' => $form->createView()];
         }
 
-        if (!$this->getVettingService()->isLoaSufficientToStartProcedure($command)) {
+        if ($command->authorityLoa === null || !$this->getVettingService()->isLoaSufficientToStartProcedure($command)) {
```

Comment:
I think moving the null check to: `\Surfnet\StepupRa\RaBundle\Service\VettingService::isLoaSufficientToStartProcedure` is better from a maintainability point of view. Also I think the service should verify if the command is configured correctly, not the controller.

Reply:
And consider adding a test case to: `\Surfnet\StepupRa\RaBundle\Tests\Service\VettingServiceTest`

Reply:
@MKodde i think its more important to find the rootcause. Based on the saml token a nullable loa should not occur.